packages/perseus/src/widgets/interactive-graphs/graphs/screenreader-text.ts

```diff
@@ -0,0 +1,12 @@
+const numberFormat = new Intl.NumberFormat(
+    undefined, // use default locale
```

Comment:
I'm not totally sure this is what we want. We might need to use the `locale` from the `PerseusI18nContext`.